authentication/authentication-internal-service/src/main/java/io/mosip/authentication/internal/service/controller/HotlistEventController.java

```diff
@@ -98,4 +120,31 @@ private boolean validateHotlistEventData(Map<String, Object> hotlistEventData) {
 								&& hotlistEventData.get(EXPIRY_TIMESTAMP) instanceof String));
 	}
 
+	
+	@PreAuthorize("hasAnyRole('REGISTRATION_PROCESSOR')")
+	@GetMapping(path = "/hotlist/status/id/{id}/idtype/{idtype}", produces = MediaType.APPLICATION_JSON_VALUE)
+	public ResponseEntity<HotlistResponseDTO> getHotlistingStatus(@PathVariable String id, @PathVariable String idtype) throws IdAuthenticationAppException{
+		HotlistResponseDTO response = new HotlistResponseDTO();
+		response.setId(API_ID);
+		response.setVersion(API_VERSION);
+		response.setResponseTime(getResponseTime());
+		try {
+			HotlistDTO hotlistStatus = hotlistService.getHotlistStatus(id, idtype);
+			response.setResponse(hotlistStatus);
+			return new ResponseEntity<>(response,HttpStatus.OK);
+		}
+		catch (IdAuthenticationBusinessException e) {
+			logger.error(IdAuthCommonConstants.SESSION_ID, e.getClass().toString(), e.getErrorCode(), e.getErrorText());
+			throw new IdAuthenticationAppException(e.getErrorCode(), e.getErrorText(), e);
+		}
+	}
+ 
+	private String getResponseTime() {
+		return DateUtils.formatDate(
```

Comment:
I think you can directly use the Date string from below code instead of converting the string to data and then converting back to string:
```
DateUtils.formatToISOString(DateUtils.getUTCCurrentDateTime()),
						environment.getProperty(IdAuthConfigKeyConstants.DATE_TIME_PATTERN),
						TimeZone.getTimeZone(ZoneOffset.UTC))
```

Reply:
Please check the above

Reply:
yes i will change it